packages/components/tests/dummy/app/templates/components/badge.hbs

```diff
@@ -102,6 +102,23 @@
       <Hds::Badge @icon="activity" @text="Lorem ipsum" /></p>
   </div>
 
+  <h5 class="dummy-h5">Containers</h5>
+  <div class="dummy-form-text-input-containers">
```

Comment:
always better to copy&paste these helper classes from one component page to another (in this case, the class `dummy-form-text-input-containers` is specific to the `text-input` doc page).

_Notice: once we will start to work on the "proper" website, we will also abstract these classes but for now copy&paste between the `db-[component].scss` files is OK_

Reply:
I copied the markup without realising it was specific to the text-input component 😬 
Have updated with porting these classes to the `db-badge.scss` – and we'll abstract at a further point, as you said.